File: src/csharp/Intel/Iced/Intel/FastFormatterInternal/Registers.cs
// SPDX-License-Identifier: MIT
// Copyright (C) 2018-present iced project and contributors

#if FAST_FMT
using Iced.Intel.FormatterInternal;

namespace Iced.Intel.FastFormatterInternal {
	static class Registers {
		public static readonly FormatterString[] AllRegisters = RegistersTable.GetRegisters();
	}
}
#endif

File: src/rust/iced-x86/src/formatter/intel/tests/registers.rs
// SPDX-License-Identifier: MIT
// Copyright (C) 2018-present iced project and contributors

use crate::formatter::intel::tests::fmt_factory;
use crate::formatter::tests::registers::register_tests;

#[test]
fn test_regs() {
	register_tests("Intel", "RegisterTests", || fmt_factory::create_registers());
}
